src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/adyen/partialPayment.js

```diff
@@ -0,0 +1,65 @@
+const Logger = require('dw/system/Logger');
+const Transaction = require('dw/system/Transaction');
+const Money = require('dw/value/Money');
+const AdyenConfigs = require('*/cartridge/scripts/util/adyenConfigs');
+const adyenCheckout = require('*/cartridge/scripts/adyenCheckout');
+const AdyenHelper = require('*/cartridge/scripts/util/adyenHelper');
+
+function makePartialPayment(req, res, next) {
+  try {
+    const request = JSON.parse(req.body);
+
+    const { paymentMethod, partialPaymentsOrder, amount } = request;
+
+    const partialPaymentRequest = {
+      merchantAccount: AdyenConfigs.getAdyenMerchantAccount(),
+      amount,
+      reference: 'partialPaymentRef',
```

Comment:
Should we find a suitable value for this? maybe something tying to a basket or something else? (I guess we do not have an order at this stage...)